runtime/moonbeam/src/xcm_config.rs

```diff
@@ -485,8 +485,6 @@ pub enum CurrencyId {
 	SelfReserve,
 	// Assets representing other chains native tokens
 	ForeignAsset(AssetId),
-	// Our local assets
-	DeprecatedLocalAssetReserve(AssetId),
```

Comment:
Please keep a dummy variant here, youc an remove the (AssetId) parameter and use an empty varient named `RemovedLocalAssetVariant`